aspnetcore/signalr/hubcontext/sample/SignalRNotify.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Even though this is now implicit, I'd prefer being explicit in a sample

Reply:
>I'd prefer being explicit in a sample

The version number I'm assuming?